turbo.json

```diff
@@ -25,6 +25,10 @@
             "outputs": ["dist/**"],
             "dependsOn": ["@elizaos/plugin-node#build"]
         },
+        "@elizaos/plugin-evm#build": {
```

Comment:
Added a build step for the plugin-evm, which is crucial for the integration process.